### DISCUSSION TITLE
[Question]: 请问怎么得出的“can generate up to 8K tokens ”这个结论？可以描述下具体的评估方法吗？

### DISCUSSION THREAD

**Discussion by kakaxisisan** [QwenLM/Qwen3#966] [Q&A]
### Has this been raised before?

- [X] I have checked [the GitHub README](https://github.com/QwenLM/Qwen2.5).
- [X] I have checked [the Qwen documentation](https://qwen.readthedocs.io) and cannot find an answer there.
- [X] I have searched [the issues](https://github.com/QwenLM/Qwen2.5/issues?q=is%3Aissue) and there is not a similar one.
- [X] I confirm that this is not a bug report, a feature request, or a badcase.

### Description

如题

**Comment by slatter666** [comment]
Is there anyone can answer this question? It support 128K context, why 8k generation?
